homeassistant/helpers/intent.py

```diff
@@ -138,15 +138,61 @@ def _has_name(
     if name in (state.entity_id, state.name.casefold()):
         return True
 
-    # Check aliases
-    if (entity is not None) and entity.aliases:
-        for alias in entity.aliases:
-            if name == alias.casefold():
-                return True
+    # Check name/aliases
+    if entity is not None:
```

Comment:
This is code from the other PR that we weren't pursueing right now? Let's remove it as it's not correctly handling entity registry names.

Reply:
Removed